smart_contracts.rst

```diff
@@ -554,7 +556,7 @@ Called after a channel has been closed. Can be called by any Ethereum address an
 - ``non_closing_participant``: Ethereum address of the channel participant who is updating the balance proof data.
 - ``balance_hash``: Hash of the balance data
 - ``nonce``: Strictly monotonic value used to order transfers.
-- ``additional_hash``: Computed from the message. Used for message authentication.
+- ``additional_hash``: Computed from the offchain message. Used for message authentication.
```

Comment:
Perhaps add also here the same note: That this hash can in the future also contain other application specific meta data?

Reply:
Done.